modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -108,6 +211,69 @@ NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[reque
 If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
 
 
+=== Real-time Collaboration
+
+The following improvements, changes, and fixes were made to Real-time Collaboration.
+
+Three additional options are now supported when Real-time Collaboration is active:
+
+* `valid_elements`
+* `valid_children`
+* `extended_valid_elements`
+
+Real-time Collaboration now supports the `autocompleter()` API.
+
+//xref:autocompleter.adoc[`autocompleter()`] ?
+
+Real-time Collaboration now supports the `inline_boundaries` option.
+
+// xref:partial$configuration/inline-boundaries.adoc[`inline_boundaries`] ?
```

Comment:
This isn't ideal — the docs stylesheet doesn't show links in code blocks other than on hover — but these are the links I think we should use.

```suggestion
Real-time Collaboration now supports the xref:autocompleter.adoc[`autocompleter`] API.

Real-time Collaboration now supports the xref:content-behavior-options.adoc#inline_boundaries[`inline_boundaries`] option.
```

Reply:
In terms of making the links more visible, would something like this work better?

+ Real-time Collaboration now supports xref:autocompleter.adoc[the `autocompleter` API].
+ 
+ Real-time Collaboration now supports xref:content-behavior-options.adoc#inline_boundaries[the `inline_boundaries` option].

Reply:
yeah that would be better :)

Reply:
Going with the better approach. :-)